.github/workflows/buildx.yml

```diff
@@ -0,0 +1,147 @@
+    strategy:
+      # allowing fails ..
+      fail-fast: false
+      matrix:
+        # -3.3 debian for linux/arm64
+        postgres: [11, 12, 13, 14, 15]
+        postgis: ['3.3']
+        variant: [default]
+        platforms: ['linux/arm64']
+        include:
+          - postgres: 10
+            postgis: '3.2'
+            variant: default
+            platforms: 'linux/arm64'
+
+        # 15-3.3-alpine for every platforms
+          - postgres: 15
+            postgis: '3.3'
+            variant: 'alpine'
+            platforms: 'linux/arm64'
+          - postgres: 15
+            postgis: '3.3'
+            variant: 'alpine'
+            platforms: 'linux/arm/v7'
+          - postgres: 15
+            postgis: '3.3'
+            variant: 'alpine'
+            platforms: 'linux/arm/v6'
+          - postgres: 15
+            postgis: '3.3'
+            variant: 'alpine'
+            platforms: 'linux/386'
+          - postgres: 15
+            postgis: '3.3'
+            variant: 'alpine'
+            platforms: 'linux/ppc64le'
+
+         # 15-3.3 debian for linux/ppc64le
+          - postgres: 15
+            postgis: '3.3'
+            variant: 'default'
+            platforms: 'linux/ppc64le'
+
+    name: ${{ matrix.platforms }} ${{ matrix.postgres }}-${{ matrix.postgis }} ${{ matrix.variant }}
+    runs-on: ubuntu-latest
+
+    env:
+      VERSION: ${{ matrix.postgres }}-${{ matrix.postgis }}
+      VARIANT: ${{ matrix.variant }}
+      # Not the default (main) workflow!
+      WORKFLOW: buildx
+      # Docker image tag postfix
+      TAG_POSTFIX: '-experimental'
+
+    steps:
+
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: "debug: docker buildx inspect --bootstrap"
+        run:  docker buildx inspect --bootstrap
+
+      - name: "debug: docker buildx ls"
+        run: docker buildx ls
+
+      - name: "debug: Inspect the base alpine image postgres:${{ matrix.postgres }}-alpine3.16"
+        if: ${{ env.VARIANT == 'alpine' }}
+        run: docker pull --platform=${{ matrix.platforms }} postgres:${{ matrix.postgres }}-alpine3.16 && docker inspect postgres:${{ matrix.postgres }}-alpine3.16
+        env:
+          DOCKER_DEFAULT_PLATFORM: ${{ matrix.platforms }}
+          PLATFORM: ${{ matrix.platforms }}
+
+      - name: "Build/Test: ${{ env.VERSION }} ${{ env.VARIANT }}${{env.TAG_POSTFIX}} ${{ env.DOCKER_DEFAULT_PLATFORM }}"
+        run: make test
+        env:
+          DOCKER_DEFAULT_PLATFORM: ${{ matrix.platforms }}
+          PLATFORM: ${{ matrix.platforms }}
+          # Buildx/qemu test need more time
+          POSTGRES_TEST_TRIES: 42
+          POSTGRES_TEST_SLEEP: 2
+
+      - name: "debug: docker images"
+        run: docker images
+
+      - name: "debug: docker inspect postgis/postgis:${{ env.VERSION }}${{ env.TAG_POSTFIX }}"
```

Comment:
this instances of postgis/postgis can be defined as REPO and IMAGE variables to avoid hard coding?  This is more important for testability.

Reply:
Done; 
Added "REPO_NAME" ;  "IMAGE_NAME"

example:
```
 env:
   ALPINE_VERSION: 3.16
   WORKFLOW: buildx
   REPO_NAME: postgis
   IMAGE_NAME: postgis
   VERSION: 15-3.3
   VARIANT: alpine
   TAG_POSTFIX: -experimental
```